src/main/java/net/mcreator/ui/workspace/breadcrumb/WorkspaceFolderBreadcrumb.java

```diff
@@ -100,7 +104,7 @@ public void reloadPath(FolderElement file, Class<? extends IElement> childElemen
 
 			add(entry);
 
-			if (idx < path.size() - 1 || alwaysSuggestChoose) {
+			if (selectionCondition == null || selectionCondition.test(filePathPart)) {

```

Comment:
This is even more complex. Please revert this. This could be simple parameter with simple name, not this complex system

Reply:
`boolean canExpandTrailHead`